plugins/modules/proxysql_replication_hostgroups.py

```diff
@@ -278,17 +283,60 @@ def delete_repl_group(self, check_mode, result, cursor):
                              " mysql_replication_hostgroups, however" +
                              " check_mode is enabled.")
 
+    def update_check_type(self, cursor):
+        try:
+            query_string = \
+                """UPDATE mysql_replication_hostgroups
+                    SET check_type = %s
+                    WHERE writer_hostgroup = %s
+                """
+
+            query_data = \
+                [self.check_type, self.writer_hostgroup]
+
+            cursor.execute(query_string, query_data)
+        except Exception as e:
+            pass
+
+    def update_reader_hostgroup(self, cursor):
+        query_string = \
+            """UPDATE mysql_replication_hostgroups
+                SET reader_hostgroup = %s
+                WHERE writer_hostgroup = %s
+            """
+
+        query_data = \
+            [self.reader_hostgroup, self.writer_hostgroup]
+
+        cursor.execute(query_string, query_data)
+
+    def update_comment(self, cursor):
+        query_string = \
+            """UPDATE mysql_replication_hostgroups
+                SET comment = %s
+                WHERE writer_hostgroup = %s
+            """
+
+        query_data = \
+            [self.comment, self.writer_hostgroup]
+
+        cursor.execute(query_string, query_data)
```

Comment:
```suggestion
        cursor.execute(query_string, (self.comment, self.writer_hostgroup))
```
just a suggestion. imo the function invocation is still pretty short. though it's a question of personal taste